src/audio/codec_adapter/CMakeLists.txt

```diff
@@ -13,6 +13,4 @@ endif()
 if(CONFIG_WAVES_CODEC)
 add_local_sources(sof codec/waves.c)
 sof_add_static_library(MaxxChrome ${CMAKE_CURRENT_LIST_DIR}/lib/release/libMaxxChrome.a)
-# folder with Waves API must be among include directories
-target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/codec_adapter/codec/)
```

Comment:
SOF with Waves codec will not compile with those changes
as a comment says Waves API folder should be on includes path
new API location is not on includes path
so do not delete this but rather change it to 
`target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)`
and keep a comment